content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx

```diff
@@ -252,7 +252,7 @@ The following constraints apply when you combine `groupBy` and `orderBy`:
 
 #### Order by aggregate group
 
-You can **order by aggregate group**. Prisma added support for using the ORDER BY clause with aggregated groups in relational databases in version [2.21.0](https://github.com/prisma/prisma/releases/2.21.0) and support for MongoDB in [3.3.4](https://github.com/prisma/prisma/releases/3.4.0). 
+You can **order by aggregate group**. Prisma added support for using `orderBy with aggregated groups in relational databases in version [2.21.0](https://github.com/prisma/prisma/releases/2.21.0) and support for MongoDB in [3.4.0](https://github.com/prisma/prisma/releases/3.4.0). 
```

Comment:
The closing ` is missing unfortunately which probably messes with formatting.

Reply:
Surprisingly (to me) does not, just does not format the `orderBy` as intended: 
![image](https://user-images.githubusercontent.com/183673/140051031-0955af1c-c901-47e3-92b9-caed4deace40.png)